src/ip/Address.h

```diff
@@ -34,12 +35,41 @@
 namespace Ip
 {
 
+class Address;
+
```

Comment:
This forward declaration should be provided by ip/forward.h.

Reply:
it is. Removing.